pennylane/ops/op_math/controlled.py

```diff
@@ -866,3 +878,30 @@ def parameter_frequencies(self):
             "and parameter frequencies can not be computed via generator for more than one "
             "parameter."
         )
+
+
+# Program capture with controlled ops needs to unpack and re-pack the control wires to support dynamic wires
+# See capture module for more information on primitives
+if Controlled._primitive is not None:  # pylint: disable=protected-access
```

Comment:
Is this because importing may pass this position before there is a `_primitive` registered? :thinking: Would be good to comment on that as well :)

Reply:
Adding a source code comment to this effect. If `Controlled._primitive`, then jax isn't installed so the `_primitive` was never created and set.